CHANGELOG-2.1.md

```diff
@@ -1,16 +0,0 @@
-# Changelog since v2.0.1
```

Comment:
we should keep this file?

Reply:
oh I only saw 1 file so I assumed we were doing replacements

Reply:
fixed